content/MinimumCD/signatures.md

```diff
@@ -7,7 +7,7 @@ weight: 2
 Do you want to become a signatory? Read our [contribution guidelines](https://github.com/Minimum-CD/cd-manifesto/blob/master/CONTRIBUTING.md).
 
 {{%excerpt%}}
-
+[Garrard&nbsp;Kitchen](https://www.linkedin.com/in/garrard-kitchen/)&nbsp;-
```

Comment:
Please move signature to the end of the file

Reply:
done